pkg/scheduler/preemption/preemption.go

```diff
@@ -96,16 +97,25 @@ func (p *Preemptor) GetTargets(wl workload.Info, assignment flavorassigner.Assig
 	if len(sameQueueCandidates) == len(candidates) {
 		// There is no possible preemption of workloads from other queues,
 		// so we'll try borrowing.
-		targets = minimalPreemptions(&wl, assignment, snapshot, resPerFlv, candidates, true)
+		targets = minimalPreemptions(&wl, assignment, snapshot, resPerFlv, candidates, true, nil)
```

Comment:
let's just return the targets here to reduce indentation.

Reply:
Done